tests/UI/campaigns/functional/BO/15_header/01_leftMenu.ts

```diff
@@ -1,23 +1,20 @@
 // Import utils
 import helper from '@utils/helpers';
+import loginCommon from '@commonTests/BO/loginBO';
 
 // Import test context
 import testContext from '@utils/testContext';
 
-require('module-alias/register');
-
-const {expect} = require('chai');
-
-// Import utils
-const loginCommon = require('@commonTests/BO/loginBO');
-
 // Import pages
-const dashboardPage = require('@pages/BO/dashboard');
+import dashboardPage from '@pages/BO/dashboard';
+
+import {expect} from 'chai';
+import {BrowserContext, Page} from 'playwright';
 
 const baseContext = 'functional_BO_header_leftMenu';
 
-let browserContext;
-let page;
+let browserContext: BrowserContext;
+let page: Page;
```

Comment:
not related but, what about initializing these variables inside the `describe` scope itself? If there are no reason to initialize them globally, maybe it could avoid some side effects later

Reply:
Change done. Thanks for your informations.